src/connectors/stripe/index.ts

```diff
@@ -246,6 +246,24 @@ class StripeService {
       this.handleError(error)
     }
   }
+
+  /**
+   * Get customer portal URL
+   */
+  getCustomerPortal = async ({ customerId }: { customerId: string }) => {
+    try {
+      if (!environment.stripeReturnURL) {
+        throw new ServerError('matters stripe return URL has not been set')
+      }
+      const session = await this.stripe.billingPortal.sessions.create({
+        customer: customerId,
+        return_url: environment.stripeReturnURL,
```

Comment:
I saw `return_url` is optional and can be configured in [Stripe Dashboard](https://dashboard.stripe.com/settings/billing/portal), would it more simple if use the latter instead?

Reply:
@robertu7 the `return_url` configuration is removed from the code as requested. pls check.